test/oli/accounts_test.exs

```diff
@@ -104,6 +104,63 @@ defmodule Oli.AccountsTest do
 
       assert author.preferences.timezone == "America/Los_Angeles"
     end
+
+    test "update author password successfully" do
+      author = insert(:author)
+
+      attrs = %{
+        "current_password" => "password_1",
+        "email" => author.email,
+        "family_name" => author.family_name,
+        "given_name" => author.given_name,
+        "password" => "password_2",
+        "password_confirmation" => "password_2"
+      }
+
+      assert {:ok, _author = %Author{}} = Accounts.update_author(author, attrs)
+    end
+
+    test "update author password fails when new password and password confirmation are different" do
+      author = insert(:author)
+
+      attrs = %{
+        "current_password" => "password_1",
+        "email" => author.email,
+        "family_name" => author.family_name,
+        "given_name" => author.given_name,
+        "password" => "password_22",
+        "password_confirmation" => "password_2"
+      }
```

Comment:
What do you think of shrinking this map to only have the concerning pieces of data: `current_password`, `password` and `password_confirmation`?The others parts can be skipped because we are added by the factory.